curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/set-up-the-environment.english.md

```diff
@@ -37,7 +37,7 @@ Submit your page when you think you've got it right.
 tests:
   - text: Socket.IO is a dependency
     testString: getUserInput => $.get(getUserInput('url')+ '/_api/package.json') .then(data => { var packJson = JSON.parse(data); assert.property(packJson.dependencies, 'socket.io', 'Your project should list "socket.io" as a dependency'); }, xhr => { throw new Error(xhr.statusText); })
-  - text: Socket.IO has been properly required and instanciated
+  - text: Socket.IO has been properly required and instantiated
```

Comment:
While we are fixing this, let's go ahead and make it match the assert message argument, because the `text` is actually what displays now plus add the code highlighting.
```suggestion
  - text: You should correctly require and instantiate <code>socket.io</code> as <code>io</code>.
```